js/thankyou.js

```diff
@@ -27,6 +27,7 @@
 'Go raibh maith agat as do chuid ranníocaíochtaí!',
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
+'സംഭാവന ചെയ്തതിന് നന്ദി',
```

Comment:
Wrong spelling :P

Reply:
Do u mean I Have written it wrong in Malayalam

Reply:
Yes